data/ship builders jobs.txt

```diff
@@ -0,0 +1,132 @@
+			label "help request"
+			`	Ms. Oxygenpocket pauses for a moment and then continues; "We have an Hauler III that we upgraded with the cargo-handling systems for the VI so that we can test those new systems.`
+			`Normally our own department would escort such prototypes, but there is a mole in our organisation and we have very strong competition from the Syndicated Shipyards.`
+			`I only need you to help us travel to the nearby planet Skymoot to deliver sushi for a formal dinner. You will be paid for your expenses.`
+			`Will you help us?"`
+			choice
+				`	"Isn't it dangerous to transport food in an untested prototype?`
+					goto "risky transport"
+				`	"Sure, I'll help you."`
+					accept
+				`	"Sorry, I don't have time to help you."`
+					decline
+			label "risky transport"
+			`	"No it's actually more of a formality. Those systems have gone through a lot of simulation cycles already."`
+			choice
+				`	"I'll help you."`
+					accept
+				`	"Sorry, I don't have time to help you."`
+					decline
+	on complete
+		dialog
+			`You are waiting for the Hauler prototype to land, but it appears to stay in orbit.`
+			`There is no response to your hails by radio. Looks like you need to take off and investigate.`
+
+
+mission "Escort Hauler prototype to student party"
+	description "Escort the ship with the new cargo-systems to <destination> to allow it to deliver cooked fish dishes to an archeology students party."
+	source "Skymoot"
+	destination "Zug"
+	landing
+	to offer
+		has "Escort Hauler prototype to formal dinner: offered"	
+	npc accompany
+		government "Merchant"
+		ship "Hauler III" "SB-Labs Cargo Cooker"
+	on enter
+		dialog
+			`As soon as you take off you receive an angry voice message over a short-range inter-ship communications channel: "Why were you trying to contact me over radio! Does everybody here need to know that we are flying around here???`
+			`Let's go to <destination>." You wonder why the destination changed.`
+	on complete
+		dialog
+			`This time the Hauler prototype appears to be landing right behind you, but the ship executes a go-around just before touching the landing pad and flies away at full thrust.`
+			`You receive a short text-message over the short-range inter-ship communiation channel: "The colleagues at the shipyard cannot see us like this!"`
+
+mission "Escort Hauler prototype to factory"
+	description "Escort the ship with the new cargo-systems to <destination> to allow it to deliver overcooked fish to a floating catfood factory."
+	source "Zug"
+	destination "Mere"
+	landing
+	to offer
+		has "Escort Hauler prototype to student party: offered"
+	npc accompany
+		government "Merchant"
+		ship "Hauler III" "SB-Labs Cargo Processor"
+	on complete
+		dialog
+			`The Hauler VI prototype descends at full thrust and lands way ahead of you on a landing pad from a floating catfood factory.`
```

Comment:
"Descends at full thrust" is a weird phrase. So is "on a landing pad from a ... factory."